templates/taxi/index.html

```diff
@@ -9,5 +9,6 @@ <h2>Dynamic content</h2>
     <li><strong>Cars:</strong> {{ num_cars }}</li>
     <li><strong>Drivers:</strong> {{ num_drivers }}</li>
     <li><strong>Manufacturers:</strong> {{ num_manufacturers }}</li>
+    <li><strong>Visit counter:</strong> {{ num_visits }}</li>
```

Comment:
Add this part according to the task screenshot. It would be a good practice to utilize the `pluralize` from the checklist. Add an updated screenshot with this part.